src/plugins/intel_cpu/src/nodes/paged_attn.cpp

```diff
@@ -165,7 +165,12 @@ void PagedAttention::createPrimitive() {
 
         size_t key_group_size = cpuConfig.keyCacheGroupSize;
         size_t value_group_size = cpuConfig.valueCacheGroupSize;
-        return make_pa_executor(rtPrecision, kCachePrecision, vCachePrecision, key_group_size, value_group_size);
+        return make_pa_executor(rtPrecision,
```

Comment:
Do we support `key_group_size` other than 32 if by channel is enabled?

Reply:
current for by-channel quantization, `group_size` always equals to `block_size`

Reply:
Do we check the limitation of `group_size`?